types/standard/config/webpack.config.js

```diff
@@ -24,13 +24,15 @@ const buildConfig = require(path.resolve(frameworkPath, '../../build/buildConfig
 const moduleFederationPlugin = require(path.resolve(frameworkPath, '../../build/moduleFederation.js')).plugin;
 const extractThemesPlugin = require(path.resolve(frameworkPath, '../../build/themes.js')).extractThemesPlugin;
 
-const jsPath = projectConfig.jsPath;
-
+const ensureNoTrailingSlash = (path) => {
+    return path && path.endsWith("/") ? ensureNoTrailingSlash(path.slice(0, -1)) : path;
+}
+const jsPath = ensureNoTrailingSlash(projectConfig.jsPath);

```

Comment:
Why we should remove all training slashes here?
I think the problem is caused by the chunks paths, generated using "//" . 
Maybe I don't see it in the geonode context.